demo/script-tag.js

```diff
@@ -1,11 +1,11 @@
-var clearDrawing = function() {
+var clearDrawing = function () {
```

Comment:
This isn't a blocker for this commit, but for future work it would be good to see if we can use a linter or something to avoid things like this, as it makes the diff very messy for poor reviewers trying to see what the important changes are :-)

Reply:
@odscjames This is the recommended change from eslint/prettier. I think Steve added some CircleCI stuff, and some linting but not sure if all the config is aligned (repo vs circleCI vs my local setup) if it didn't flag that as an issue before.I can try to wrap linting changes into a single commit when I come across them though.

Reply:
Cool - that's grand, it's just making sure that linter is consistently used then. Personally I'd put it in a GH action that must pass before a PR, then lint everything in one commit/PR. Like we do for Python/Black. Then this won't happen again. I'm not a fan of linting as changes are made - that just means there are lots of linting commits! Just do it all in one big commit then never have to worry about it again, I would say.

Reply:
+1 I need to find some time to look at the pipeline stuff sometime